### PR TITLE
new way to start with a new strategy

### PR DESCRIPTION
**Pull request by marioarranzr**
I found a bit complicated to copy and paste every strategy into `main.go` and I think this is a simpler solution when the moment of switching strategies comes.

I basically moved the examples of every type of strategy to a new package `examples` and, instead having to copy and paste the available strategies into `main.go`
```
var myStrategy = WebsocketStrategy{...} // old way
bot.AddCustomStrategy(myStrategy)
```
we define now the strategies in the `examples` package as functions
```
func MyStrategy() strategies.Strategy {...}
```
and the ones we need to have available are added in `main.go` 
```
bot.AddCustomStrategy(examples.MyStrategy()) // new way
```

### CODE REVIEW

**Comment by saniales** [comment]
Yeah this is a cool way to handle examples, but why using functions instead of using global variables?
Since these are examples if they are edited they become personal and should be handled in another (personal) package which imports the framework

**Comment by marioarranzr** [comment]
I used functions instead of variables because I considered some of them can be added and others not and it is as easy as call only the functions we need to import to the system.
Sure, we can use `personal` package instead `example`.

**Comment by saniales** [comment]
ok I will not merge it into this branch, however I approve the examples package. Avoid functions, use global variables and I will accept your PR :)

**Comment by saniales** [comment]
@marioarranzr you can also consider to contribute to the wiki as well :) you can add new chapters (I will take care of the linking to the main content) or edit existing ones

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ If you need to, you can create a strategy and bind it to the bot:
 import bot "github.com/saniales/golang-crypto-trading-bot/cmd"
 
 func main() {
-    bot.AddCustomStrategy(myStrategy)
+    bot.AddCustomStrategy(examples.myStrategy())
```
Comment (saniales) [inline README.md]: see comment

Comment (marioarranzr) [reply README.md]: Done it ;)

**Comment by saniales** [comment]
I am sorry I am closing this, however I will add you as co-author when I fix this in one of my commits, thanks for contributing. I will close this PR when this will be fixed by me :)

**Comment by marioarranzr** [comment]
Is there any reason? I think I fixed what you said, right?

**Comment by saniales** [comment]
The problem is that we can resume everything by just changing the folder and moving examples and this is only one commit, if you can just move the files to a example folder and fix references I will accept the PR 
this is to avoid to fill history with garbage commits :)

**Comment by marioarranzr** [comment]
When merge it only creates a single commit with the name of the PR

File: examples/websocket.go [hunk 2]
```diff
@@ -22,8 +22,12 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// Websocket strategy
-func Websocket() strategies.Strategy {
+var (
+	Websocket = websocket()
```
Comment (saniales) [inline examples/websocket.go]: why using this if it's not called twice? just leave original assignment as it was before
``` go
strategies.WebsocketStrategy{
		Model: strategies.StrategyModel{
			Name: "Websocket",
			Setup: func(wrappers []exchanges.ExchangeWrapper, markets []*environment.Market) error {
				for _, wrapper := range wrappers {
					err := wrapper.FeedConnect(markets)
					if err == exchanges.ErrWebsocketNotSupported || err == nil {
						continue
					}
					return err
				}
				return nil
			},
			OnUpdate: func(wrappers []exchanges.ExchangeWrapper, markets []*environment.Market) error {
				// do something
				return nil
			},
			TearDown: func(wrappers []exchanges.ExchangeWrapper, markets []*environment.Market) error {
				return nil
			},
			OnError: func(err error) {
				logrus.Error(err)
			},
		},
	}
```

**Comment by saniales** [comment]
See comment and apply it to all examples, then I will merge squash :)

File: examples/websocket.go (hunk 2 above)
Comment (marioarranzr) [reply examples/websocket.go]: I created all variables calling to their functions
```
var (
	Watch5Sec                  = watch5Sec()
	SlackIntegrationExample    = slackIntegrationExample()
	TelegramIntegrationExample = telegramIntegrationExample()
)
``` 
and 
```
var (
	Websocket = websocket()
)
```
Just trying to be tidy

Comment (marioarranzr) [reply examples/websocket.go]: Maybe should be `WebsocketExample = websocketExample()` and the same with `watch5Sec`

Comment (saniales) [reply examples/websocket.go]: I dont want that, since we refer only to the global variable and use functions only once

**Pull request merged**